Mathlib/Analysis/SpecificLimits/Normed.lean

```diff
@@ -229,6 +229,21 @@ theorem tendsto_pow_const_mul_const_pow_of_abs_lt_one (k : ℕ) {r : ℝ} (hr :
   rw [tendsto_zero_iff_norm_tendsto_zero]
   simpa [div_eq_mul_inv] using tendsto_pow_const_div_const_pow_of_one_lt k hr'
 
+/--For `k ≠ 0` and a constant `r` the function `r / n ^ k` tends to zero. -/
+lemma tendsto_const_div_pow  (r : ℝ) (k : ℕ) (hk : k ≠ 0) :
+    Tendsto (fun n : ℕ => r / n ^ k) atTop (𝓝 0) := by
+  have h := Filter.Tendsto.const_mul r (l := atTop) (f := fun (n : ℕ ) => 1 / n^k) (c := 0) ?_
+  simp only [one_div, mul_zero] at *
+  apply h.congr
+  intro y
+  ring
```

Comment:
Are you missing a focusing dot somewhere?

Reply:
yes thanks!